### PR TITLE
Make 7.2 mcrypt note more prominent and put it against mcrypt Section

### PR DESCRIPTION
**Pull request by seamuslee001**
This makes the note a bit more prominent and puts it in line with the rest of the documentation on Mcrypt extension

ping @MegaphoneJon

### CODE REVIEW

File: docs/requirements.md [hunk 1]
```diff
@@ -70,6 +68,10 @@ To install these extensions, you will typically install a separate package withi
 * [PHP SOAP](http://www.php.net/soap) - required to use the SOAP processor (In turn needed for the popular CiviSMTP service)
 * [PHP DOM XML](http://www.php.net/manual/en/dom.setup.php) - required by CiviCase
 * [PHP Mcrypt](http://php.net/manual/en/intro.mcrypt.php) - Required for SMTP credentials and other credentials in the database. If you add Mcrypt on after running CiviCRM for a while you will need to re-save your passwords to the database.
+
+    !!! warning "PHP7.2 Compatability"
```
Comment (MegaphoneJon) [inline docs/requirements.md]:
```suggestion
    !!! warning "PHP 7.2 Compatibility"
```

**Comment by MegaphoneJon** [comment]
Please leave another note once you've approved my suggestion, it doesn't give a notification so I can't re-review.  This is "merge-on-change" though.

**Comment by seamuslee001** [comment]
@MegaphoneJon made the change now

File: docs/requirements.md (hunk 1 above)
Comment (seamuslee001) [reply docs/requirements.md]: Made the change

**Pull request merged**